airbyte-integrations/connectors/source-box-data-extract/source_box_data_extract/schemas/box_file_and_text.json

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
Please don't forget to delete the boilerplate ;)

Reply:
@natikgadzhi, thank you so much for your feedback, I'll get to fix these and circle back here.